pypi_formula_mappings.json

```diff
@@ -477,9 +477,9 @@
   "tarsnapper": {
     "exclude_packages": ["six"]
   },
-  "trezor-agent": {
-    "exclude_packages": ["pillow", "six"],
-    "extra_packages": ["ledger-agent", "keepkey-agent"]
+  "termius": {
+    "package_name": "termius",
```

Comment:
Can we change the source URL to pypi one? Then we won't need to set `package_name` here

Reply:
PyPI lacks the shell completions, which we have been providing and upstream documented https://github.com/termius/termius-cli#installation